sdk/storage/azure-storage-files-datalake/swagger/README.md

```diff
@@ -259,6 +259,7 @@ directive:
       $["201"].headers["Content-Length"]["x-ms-client-name"] = "FileSize";
       $["201"].headers["Content-Length"]["x-nullable"] = true;
       $["201"].headers["x-ms-request-server-encrypted"]["x-nullable"] = true;
+      $["201"].headers["x-ms-request-server-encrypted"]["x-ms-client-default"] = "bool()";
```

Comment:
Is this superfluous? What would the code look like without this statement?

Reply:
If without this statement,  the field will be nullable like this:
```
      Nullable<bool> IsServerEncrypted;
```
In the create API,it will include `Create File` |` Create Directory` | `Rename File` | `Rename Directory`.
`Create File` |` Create Directory` scenarios return `IsServerEncrypted` to customer.
`Rename File` | `Rename Directory` scenarios don't return `IsServerEncrypted` from REST API and in the convinence layer we just return renamedClient to customer.

Reply:
What if you remove Line 261 and 262?

Reply:
It will change the code of rest_client.cpp.
before removing  Line 261 and 262, the code is:
```
      if (pRawResponse->GetHeaders().count("x-ms-request-server-encrypted") != 0)
      {
        response.IsServerEncrypted
            = pRawResponse->GetHeaders().at("x-ms-request-server-encrypted") == std::string("true");
      }
```
after removing, the code is: 
```
 response.IsServerEncrypted
          = pRawResponse->GetHeaders().at("x-ms-request-server-encrypted") == std::string("true");
```
Because `Rename` scenario doesn't return `IsServerEncrypted`, removing these lines will cause an exception.

Reply:
Ok, got it.